frontend/src/app/navigation-bar.tsx

```diff
@@ -108,92 +124,133 @@ export default function NavigationBar({}) {
         className="pb-2 pl-1 cursor-pointer"
         onClick={() => router.push("/")}
       />
-      <div className="flex-1">
+      <div className="flex-none">
         <Menu>
           <DatabaseSelector
             selectedDatabase={selectedDatabase}
             setSelectedDatabase={setSelectedDatabase}
           />
         </Menu>
+      </div>
+      <div className="flex-1 my-1 overflow-y-auto">
         {selectedDatabase && (
           <Menu>
-            <H5>Tables</H5>
-            {tables?.map((table) => (
-              <MenuItem
-                key={table.id}
-                text={table.name}
-                icon={
-                  <Icon icon={table.icon as IconName} color={table.color} />
-                }
-                onClick={() => handlePageChange(`tables/${table.id}`)}
+            <div
+              className="flex flex-row items-center gap-1 mb-1"
+              onClick={() => setTablesToggle(!tablesToggle)}
+            >
+              <H5 className="mb-0">Tables</H5>
+              <Icon
+                icon={tablesToggle ? "chevron-down" : "chevron-right"}
+                color="gray"
               />
-            ))}
-            <MenuDivider />
-            <H5>Relations</H5>
-            <MenuItem
-              icon="new-object"
-              text="New relation"
-              intent="success"
-              onClick={() => setCreateRelationToggle(true)}
-              popoverProps={{
-                usePortal: true,
-              }}
-            />
-            {relations?.map((relation) => {
-              const table1 = _.find(tables, { id: relation.table_1 });
-              const table2 = _.find(tables, { id: relation.table_2 });
-              return table1 && table2 ? (
+            </div>
+            {tablesToggle &&
```

Comment:
Should we use the blueprint collapsible component here? Smooth animation and likely better for mounting/unmounting components